content/blog/2018/08/2018-08-31-shifting-gears.adoc

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "Jenkins: Shifting Gears"
+tags:
+- development
+- core
+author: kohsuke
+---
+
+Kohsuke here. This is a message for my fellow Jenkins developers.
+
+Jenkins has been on an amazing run, but in part because we've been so good at solving a certain class of problems, the fact that we've been unable to solve other classes of problems are getting more pronounced. I believe we are trapped in a local optimum, and losing appeal to people who fall outside of our traditional sweet spot.
+
+I say it's time we tackle these problems head on. I've been talking to various folks, and I think we need to take on two initiatives. One is what I call "Cloud Native Jenkins," and the other is to insert a jolt in Jenkins. As CTO of CloudBees, we are committed to solving these problems, but this will not succeed without your participation and us collaborating.
+
+Please read on https://docs.google.com/document/d/1yE9kOtlhc2vQByS6P-WgPFPu0vOG0Myb8PtqTsUxZMY/edit[my full write-up], or watch the video below. Slides I've used are https://drive.google.com/open?id=160LjRKw42XXuXT2n4j8BLCzsSL5z2m2_[here].
```

Comment:
Would it make sense to just add the writeup text to the blogpost? It would make the blogpost meaty so that people get real content without following the link

Reply:
@oleg-nenashev  That's my thought as well.  I'm pulling it in and will submit an update.

Reply:
@bitwiseman OK, thanks

Reply:
It's up to you Liam, but on 2nd thought, I'm pretty sure Google Doc is much better formatted to read text that long. And at the same time, it's so long not everyone is going to read it.

I think the purpose the blog post is to act as a teaser, and to act as the summary for busy people. From that perspective it makes more sense to keep the blog post short, and invite readers to one of the several formats that present the content.